fetch.bzl

```diff
@@ -222,3 +222,30 @@ genrule(
         ],
         sha256 = "d7c7af5d86f43a885069408a89788f67f248e8124c682bb73936f33874e0611b",
     )
+
+    http_file(
```

Comment:
I should put rules_multitool in here as devdep

Reply:
that'd be great a follow-up